python/lsst/pipe/analysis/utils.py

```diff
@@ -2041,3 +2153,52 @@ def calcQuartileClippedStats(dataArray, nSigmaToClip=3.0):
         clipValue=clipValue,
         goodArray=good,
     )
+
+
+def corr2(ra, dec, g1a, g2a, g1b=None, g2b=None, ra_units="degrees", dec_units="degrees", **treecorr_kwargs):
+    """ Function to compute auto-/cross-correlations between atmost two shear-like fields.
+    This is used to compute Rho statistics, given the appropriate shear-like fields.
+
+    Parameters
+    ----------
+    ra : `numpy.array`
+       The right ascension values of entries in the catalog
+    dec : `numpy.array`
+       The declination values of entres in the catalog
+    g1a : `numpy.array`
+       The first component of the primary shear-like field
+    g2a : `numpy.array`
+       The second component of the primary shear-like field
+    g1b : `numpy.array`, optional
+       The first component of the secondary shear-like field.
+       Autocorrelation of the primary field is computed if None (None by default)
+    g2b : `numpy.array`, optional
+       The second component of the secondary shear-like field.
+       Autocorrelation of the primary field is computed if None (None by default)
+    ra_units : `str`, optional
```

Comment:
As mentioned previously please be consistent with cases, it should be consistent with the rest of utils.py and also with the rest of this function (catA for example). It is preferable to be consistent with the rest of the file you are editing rather than external packages.